packages/zero-runtime/src/styled.d.ts

```diff
@@ -1,15 +1,116 @@
-// @TODO - Implement correct style definitions
+import type * as React from 'react';
```

Comment:
The styles here make sense, however, it's not possible to test this without tests for the types. I would recommend copying over the `styled.spec.tsx` file from the @mui/material, adjusting them and seeing what fails.

Reply:
Done.